src/ui/common/src/utils/integrations.ts

```diff
@@ -429,6 +436,11 @@ export const SupportedIntegrations: ServiceInfoMap = {
     category: IntegrationCategories.NOTIFICATION,
     docs: `${addingIntegrationLink}/connecting-to-slack`,
   },
+  ['Spark']: {
+    logo: ServiceLogos['Spark'],
+    activated: true,
```

Comment:
Will change this to 'false' until full functionality is added!

Reply:
👍